scripts/build/utils.js

```diff
@@ -66,6 +66,11 @@ async function exec(cmd, args, options = {}) {
 }
 exports.exec = exec;
 
+async function execNode(args, options = {}) {
+    return exec(process.execPath, [...process.execArgv, ...args], options);

```

Comment:
Normally, you'd use fork to do this, but we have our own custom helper, so to fix the fact that we don't pass `execArgv` down, I added another.

Reply:
+